bessctl/cli.py

```diff
@@ -231,11 +238,16 @@ def list_matched(self, line, filters):
 
         ret = []
         ret_low = []
-        for cmd, score in matched_list:
-            if score == max_score:
-                ret.append(cmd)
-            else:
-                ret_low.append(cmd)
+
```

Comment:
I like the new scheme.  The two assignments right above here are redundant now...